packages/react/src/xcss-prop/index.ts

```diff
@@ -27,19 +27,31 @@ type XCSSPseudos<
     : never;
 };
 
+type XCSSMediaQuery<
+  TAllowedProperties extends keyof CSSProperties,
+  TAllowedPseudos extends CSSPseudos,
+  TAllowedMediaQueries extends string,
+  TRequiredProperties extends { requiredProperties: TAllowedProperties },
```

Comment:
Do you want a `requiredAtRules`?  (joking, but could actually be realistic one day!)

Reply:
Hehe. Yeah right now it's just assumed media queries are always available. And nothing else is (for XCSS prop at least).